packages/cli/src/commands/BaseCommand.ts

```diff
@@ -61,6 +62,12 @@ export abstract class BaseCommand extends Command {
 			this.exitWithCrash('There was an error running database migrations', error),
 		);
 
+		const pruningService = Container.get(PruningService);
```

Comment:
Should we move this to `packages/cli/src/commands/start.ts` instead after `server.start()` ?